src/presenter/general-presenter.js

```diff
@@ -14,6 +14,9 @@ export default class GeneralPresenter {
   #eventListComponent = null;
   #eventPresenters = new Map();
   #events = null;
+  #sortView = null;
+  #activeSortType = SortType.DAY;
+  #filterView = null;
   constructor({tripControlsFilters,tripEvents,pointModel}) {
```

Comment:
Это не очень покрывается критериями, но стоит внимательно следить за единым стилем в документам и делить код на логические группы (например, в других классах у тебя есть пустая строка перед конструктором)

```suggestion
  #filterView = null;

  constructor({tripControlsFilters,tripEvents,pointModel}) {
```